Dockerfile

```diff
@@ -18,7 +18,12 @@ FROM ghcr.io/apeworx/ape:${BASE_APE_IMAGE_TAG:-latest-slim}
 USER root
 COPY --from=builder /wheels /wheels
 RUN pip install --upgrade pip \
-    && pip install silverback --no-cache-dir --find-links=/wheels 
+    && pip install silverback \
+    asyncer \
+    'taskiq-sqs>=0.0.11' \
+    ape-alchemy \
+    ape-etherscan \
```

Comment:
Are these necessary to run on the cluster? or can it be user choice via `ape-config.yaml`

Reply:
@mikeshultz let me know if those are requirements.

Reply:
ape-alchemy maybe shouldn't be here, but I'm wondering if ape-etherscan should be a default plugin because if we don't have it, `Contract(address)` will not work. 

Although my memory is hazy now. Maybe an API key might also be required. Something to do with Etherscan's API protections, (maybe it's IP block based?) which is out of our control and would require a user to add their key to the env.

I mostly remember this being in stark contrast to how we run bots locally, which will probably be a huge pain point for users. Might even be worth providing a default API key.

----

All that aside, I also suggest we be fairly generous with plugins in default images. Maybe even install all of the ApeWorX developed ones. At worst it'll cost a few MB on the image with almost no downside.

Reply:
> Although my memory is hazy now. Maybe an API key might also be required. Something to do with Etherscan's API protections, (maybe it's IP block based?) which is out of our control and would require a user to add their key to the env.

I forgot if an API key makes it work, but they do block AWS IPs

Reply:
> All that aside, I also suggest we be fairly generous with plugins in default images. Maybe even install all of the ApeWorX developed ones. At worst it'll cost a few MB on the image with almost no downside.

Curious of your thoughts here but the problem with the default ape image that installs `recommended-plugins` is that it takes whatever the latest revision is of the plugins _at the moment it is built_ which may require some downstream upgrades to fix bugs or support new features present in patch builds

Reply:
> Curious of your thoughts here but the problem with the default ape image that installs `recommended-plugins` is that it takes whatever the latest revision is of the plugins _at the moment it is built_ which may require some downstream upgrades to fix bugs or support new features present in patch builds

Plugins will break silverback? Seems like minor version pins should prevent issues.

Reply:
It's only about a ~50MB difference to use the recommended-plugins. It used to be a big difference before all of the upgrades to the base image. LMK what you guys want to do here.

Reply:
> It's only about a ~50MB difference to use the recommended-plugins. It used to be a big difference before all of the upgrades to the base image. LMK what you guys want to do here.

So to play devil's advocate, get rid of `ape-slim` after all?

Reply:
@johnson2427 was having issues getting the slim to release anyways, say we just drop it

Reply:
silverback-slim will only release on version releases (I believe). I set it up the same way as ape's. Ape just didn't release the stable-slim, which I could go back to and figure out. I'm for dropping slim though.

Reply:
let's drop it, it doesn't sound useful (and in fact could be annoying)

we just have to keep in mind what the ape base images build with at the time they are built